pkg/reconciler/labeler/accessors.go

```diff
@@ -43,26 +43,26 @@ type Accessor interface {
 	makeMetadataPatch(route *v1.Route, name string, remove bool) (map[string]interface{}, error)
 }
 
-// Revision is an implementation of Accessor for Revisions.
-type Revision struct {
+// RevisionAcc is an implementation of Accessor for Revisions.
```

Comment:
I'd go with either `Revision` or `RevisionAccessor`.

Reply:
Went with full `RevisionAccessor` - I just don't like the ambiguity with v1.Revision